frontend/src/views/CellCards/components/CellCard/components/MarkerGeneTables/index.tsx

```diff
@@ -289,6 +306,11 @@ const MarkerGeneTables = ({ cellTypeId, setGeneInfoGene }: Props) => {
     };
   }, []);
 
+  // Handle cell type change, set marker genes table page back to 1
+  useEffect(() => {
+    setPage(1);
```

Comment:
random comment: Maybe worth changing page index to start from 0 instead of 1 eventually!